TODO 주석 정리

yoouyeon opened the pull request:
## 📝 관련 이슈

[MOD-55](https://moddo-kr.atlassian.net/browse/MOD-55)

## 💻 작업 내용

완료된 TODO 주석들을 제거하고, 남아있는 작업들을 정리했습니다.

남아있는 작업들은 [여기](https://www.notion.so/TODO-1e1a96b88e9780d2b6cafd57646dde63?pvs=4)에 1차적으로 정리해두었어요.

## 👻 리뷰 요구사항 (선택)

혹시 TODO로 남겨진 작업 중에 완료되거나 불필요한 작업이 있다면 알려주세요!

[MOD-55]: https://moddo-kr.atlassian.net/browse/MOD-55?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

yoouyeon committed:
remove: 미사용 usePostCreateGroupMembers 훅/API 제거

yoouyeon committed:
chore: 완료된 TODO 주석 제거 및 불필요한 파일 삭제

ongheong reviewed (approved):
넵 모두 확인했습니다! [아이콘 설정 바꾸기](https://github.com/dnd-side-project/dnd-12th-7-frontend/blob/b0f5e1f87b175d735f488c4a64ed8fe7e0ccc2ec/vite.config.ts#L34-L60) Todo를 제외하면 모두 처리해야 할 일들이네요..! 하나씩 쳐내봅시당
+ 해당 부분 주석 제거해주시면 감사하겠습니다!

yoouyeon committed:
chore: 완료된 TODO 주석 제거

https://github.com/dnd-side-project/dnd-12th-7-frontend/pull/123#pullrequestreview-2798882317

yoouyeon committed:
merge: develop branch
